package.json

```diff
@@ -137,15 +139,13 @@
         "pify": "^4.0.1",
         "nyc": "^14.1.1",
         "puppeteer": "^1.20.0",
+        "regenerator-runtime": "^0.13.3",
         "sandboxed-module": "^2.0.3",
         "surge": "^0.21.3",
+        "truffle": "^5.1.8",
         "typescript": "next",
         "underscore": "^1.9.1",
         "vinyl-source-stream": "^2.0.0",
-        "wait-port": "^0.2.6",
```

Comment:
there are unnecessary changes in this file as the wait-port and truffle dependency movement :-)

Reply:
The package.json has been normalized alphabetically in some other PR now. This isn't in the diff anymore.